vestapol/web_resources/json_resource.py

```diff
@@ -1,7 +1,11 @@
+from __future__ import annotations
 from abc import abstractmethod
-from typing import List
 from vestapol.writers import json_writer
 from vestapol.web_resources import base_resource
+from typing import Dict, List, TYPE_CHECKING
```

Comment:
```suggestion
from typing import Dict, List, Tuple, TYPE_CHECKING
```

^^ needed to type hint `data_path` below